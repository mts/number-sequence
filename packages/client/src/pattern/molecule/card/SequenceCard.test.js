import sequenceCardDefault from './SequenceCard.render'

describe('<SequenceCard />', () => {
  describe('Snaphot', () => {
    test('must match sequenceCardDefault', () => {
      expect(global.renderToJSON(sequenceCardDefault)).toMatchSnapshot()
    })
  })
})
