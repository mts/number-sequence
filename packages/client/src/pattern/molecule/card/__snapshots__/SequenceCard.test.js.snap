// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SequenceCard /> Snaphot must match sequenceCardDefault 1`] = `
<div>
  <div
    className="d-flex flex-justify-center"
  >
    <div
      className="px-3 border text-green border-blue"
    >
      1
    </div>
  </div>
  <div
    className="d-flex flex-justify-center"
  >
    <div
      className="px-3 border text-green border-blue"
    >
      1
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      1
    </div>
  </div>
  <div
    className="d-flex flex-justify-center"
  >
    <div
      className="px-3 border text-green border-blue"
    >
      1
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      2
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      1
    </div>
  </div>
  <div
    className="d-flex flex-justify-center"
  >
    <div
      className="px-3 border text-green border-blue"
    >
      1
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      3
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      3
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      1
    </div>
  </div>
  <div
    className="d-flex flex-justify-center"
  >
    <div
      className="px-3 border text-green border-blue"
    >
      1
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      4
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      6
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      4
    </div>
    <div
      className="px-3 border text-green border-blue"
    >
      1
    </div>
  </div>
</div>
`;
